machines/issuersMachine.ts

```diff
@@ -73,22 +88,36 @@ export const IssuersMachine = model.createMachine(
         invoke: {
           src: 'downloadIssuersList',
           onDone: {
-            actions: ['setIssuers'],
+            actions: ['setIssuers', 'unsetLoadingReason'],
             target: 'selectingIssuer',
           },
           onError: {
-            actions: ['setError'],
+            actions: ['setError', 'unsetLoadingReason'],
             target: 'error',
           },
         },
       },
       error: {
         description: 'reaches here when any error happens',
         on: {
-          TRY_AGAIN: {
-            actions: 'resetError',
-            target: 'displayIssuers',
-          },
+          TRY_AGAIN: [
+            {
+              description: 'not fetched issuers config yet',
+              cond: 'shouldFetchIssuersAgain',
+              actions: ['setLoadingIssuer', 'resetError'],
```

Comment:
since we are updating the loadingReason context, can we rename setLoadingIssuer to setLoadingReason.

Reply:
we are updating loading reason to be 'loading issuers' and in other areas we update loading reason to downloading credentials & so. so we are using setLoadingIssuers name for the action to differentiate loadingReason setup for different areas.

Reply:
Renamed function accordingly